lib/foreman_tasks/engine.rb

```diff
@@ -113,6 +113,7 @@ class Engine < ::Rails::Engine
         world.middleware.use Actions::Middleware::KeepCurrentUser, :before => ::Dynflow::Middleware::Common::Transaction
         world.middleware.use Actions::Middleware::KeepCurrentTimezone
         world.middleware.use Actions::Middleware::KeepCurrentRequestID
+        world.middleware.use ::Actions::Middleware::LoadSettingValues
```

Comment:
Rather than a guard in the middleware, can we only load the middleware if Foreman 2.5+ is used?